brave/brave-core
# Adds contribution date to internals page, adjusts some css

Resolves https://github.com/brave/brave-browser/issues/10137

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- claim grant
- do a tip
- go to rewards internals
- go to contribution tab
- make sure that you see created at date populated correctly

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
